docker-hub/publish/insights-analytics.md

```diff
@@ -10,17 +10,41 @@ to metrics as both raw data and summary data for a desired time span. You can
 view number of image pulls by tag or by digest, and get breakdowns by
 geolocation, cloud provider, client, and more.
 
+## View the analytics data
+
+Analytics data for your repositories is available for viewing on the Insights
+and analytics dashboard at the following URL:
+`https://hub.docker.com/orgs/{namespace}/insights`. The dashboard contains a
+chart visualization of the usage data, as well as a table where you can download
+the data as CSV files.
+
+To view data in the chart:
+
+- Select the data granularity: weekly or monthly
+- Select the time interval: 3, 6, or 12 months
+- Select one or more repositories in the list.
+
+  You can filter the list by repository name.
```

Comment:
**suggestion (non-blocking):** could we possible mention the share functionality? 🙏 I feel it's probably going to be something easily missed

Reply:
ah yeah sorry of course, I blanked on that :D